src/server-browser-scripts/seed-data.ts

```diff
@@ -0,0 +1,224 @@
+import { jsonDocs } from "./../../test/data/index";
+import shortUuid from "short-uuid";
+import { IDealTokenSwapDocument, IDealVotingSummary } from "./../entities/IDealTypes";
+import { DefaultTestAddressForSignIn } from "./../../test/data/configuration";
+import { firebaseDatabase, firebaseAuth, signInToFirebase } from "../services/firebase-helpers";
+import { collection, doc, writeBatch, getDocs, deleteDoc, addDoc, setDoc } from "firebase/firestore";
+import { IDealRegistrationTokenSwap } from "entities/DealRegistrationTokenSwap";
+
+export const DEALS_TOKEN_SWAP_COLLECTION = "deals-token-swap";
+export const PRIMARY_DAO_VOTES_COLLECTION = "primary-dao-votes";
+export const PARTNER_DAO_VOTES_COLLECTION = "partner-dao-votes";
+
+/**
+ * Creates a collection and adds a set of documents
+ * @param collectionKey
+ * @param arrayOfObjectsToAdd
+ * @returns
+ */
+export const addCollectionAndDocuments = async <T>(
+  collectionKey: string,
+  arrayOfObjectsToAdd: T[],
+) => {
+  const collectionRef = collection(firebaseDatabase, collectionKey);
+  const batch = writeBatch(firebaseDatabase);
+  arrayOfObjectsToAdd.forEach((obj) => {
+    const docRef = doc(collectionRef);
+    batch.set(docRef, obj);
+  });
+
+  return await batch.commit().catch(_ => {
+    // swallow
+  });
+};
+
+/**
+ * Clears an entire collection from the store
+ * @param collectionKey
+ */
+export const clearCollection = async (
+  collectionKey: string,
+) => {
+  const collectionRef = collection(firebaseDatabase, collectionKey);
+  const queryResult = await getDocs(collectionRef);
+  await Promise.all(queryResult.docs.map(async doc => {
+    const subCollection1 = collection(firebaseDatabase, collectionKey, doc.id, PRIMARY_DAO_VOTES_COLLECTION);
+    const subCollection2 = collection(firebaseDatabase, collectionKey, doc.id, PRIMARY_DAO_VOTES_COLLECTION);
```

Comment:
```suggestion
    const subCollection2 = collection(firebaseDatabase, collectionKey, doc.id, PARTNER_DAO_VOTES_COLLECTION);
```

![image](https://user-images.githubusercontent.com/30693990/166662604-47966899-ed86-461a-9339-c65ba11ce05f.png)